code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -352,7 +352,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/sniper_rifle
 	name = "sniper rifle"
-	desc = "The kind of gun that will leave you crying for mummy before you even realise your leg's missing"

```

Comment:
It has to deal 999 damage, to continue the trend that I suck at balance.
Remember the original acid act where all your shit turned to goo? that's me!
Fire spreading on touch? Meeee~

Also it'd be a player ref and I already have a few of those :P